packages/dds/tree/src/test/shared-tree/fuzz/fuzzUtils.ts

```diff
@@ -24,24 +24,61 @@ import {
 	forEachNodeInSubtree,
 	moveToDetachedField,
 } from "../../../core/index.js";
-import { SchemaBuilder, leaf, typedJsonCursor } from "../../../domains/index.js";
-import {
-	Any,
-	FieldKinds,
-	FlexFieldSchema,
-	type FlexTreeObjectNodeTyped,
-	type LeafNodeSchema,
-	type SchemaLibrary,
-	intoStoredSchema,
-} from "../../../feature-libraries/index.js";
-import type { ITreeCheckout, SharedTree, TreeContent } from "../../../shared-tree/index.js";
+import type { ITreeCheckout, SharedTree } from "../../../shared-tree/index.js";
 import { testSrcPath } from "../../testSrcPath.cjs";
 import { expectEqualPaths } from "../../utils.js";
+import type {
+	NodeBuilderData,
+	// eslint-disable-next-line import/no-internal-modules
+} from "../../../simple-tree/schemaTypes.js";
+import {
+	SchemaFactory,
+	TreeViewConfiguration,
+	type TreeNodeSchema,
+	type ValidateRecursiveSchema,
+} from "../../../simple-tree/index.js";
+
+const builder = new SchemaFactory("treeFuzz");
+export class GUIDNode extends builder.object("GuidNode" as string, {
+	value: builder.optional(builder.string),
+}) {}
+
+export class FuzzStringNode extends builder.object("FuzzStringNode", {
+	stringValue: builder.required(builder.string),
+}) {}
+export class FuzzNumberNode extends builder.object("FuzzNumberNode", {
+	value: builder.required(builder.number),
+}) {}
+export class FuzzHandleNode extends builder.object("FuzzHandleNode", {
+	value: builder.required(builder.handle),
+}) {}
```

Comment:
What's the motivation for having these instead of directly using string, number, and handle? I.e., why wrap them like this?

Reply:
The motivation behind this was originally to be able to call `Tree.parent` to get the field path, but decided to just use grab the `UpPath` from the anchorNode and convert that to the `DownPath` instead. 

However currently we are checking that a selected node is of type `FuzzNode` when selecting a node for the fieldEdit, and wrapping them in a node allowed made calling `Tree.is()` on the node easier, so I left it as is

Reply:
Also, if there's something awkward about `Tree.is()` and primitives then we probably need to fix that instead of getting around the issue.

Reply:
Sorry Yann, just checked that `Tree.is()` is working properly with primitives, and I was mistaken :) 

Updated the fuzz leaf types to be primitives